ideascube/utils.py

```diff
@@ -87,3 +88,7 @@ def get_file_sha256(path):
 
 def get_file_size(path):
     return os.stat(path).st_size
+
+def tag_splitter(tag_string):
+    tags = set(t.strip() for t in re.split('[;,]+', tag_string) if t.strip())
```

Comment:
Should you make this a raw-string with `r'...'`?

It wouldn't change anything with the regex you used, but if we end up modifying it in the future it might make for some hilarious debugging.